sdrbase/util/aprs.cpp

```diff
@@ -30,8 +45,9 @@ bool APRSPacket::decode(AX25Packet packet)
     if ((packet.m_type == "UI") && (packet.m_pid == "f0") && (packet.m_dataASCII.length() >= 1))
     {
         // Check destination address
-        QRegExp re("^(AIR.*|ALL.*|AP.*|BEACON|CQ.*|GPS.*|DF.*|DGPS.*|DRILL.*|DX.*|ID.*|JAVA.*|MAIL.*|MICE.*|QST.*|QTH.*|RTCM.*|SKY.*|SPACE.*|SPC.*|SYM.*|TEL.*|TEST.*|TLM.*|WX.*|ZIP.*)");
-        if (re.exactMatch(packet.m_to))
+        QRegularExpression re("^(AIR.*|ALL.*|AP.*|BEACON|CQ.*|GPS.*|DF.*|DGPS.*|DRILL.*|DX.*|ID.*|JAVA.*|MAIL.*|MICE.*|QST.*|QTH.*|RTCM.*|SKY.*|SPACE.*|SPC.*|SYM.*|TEL.*|TEST.*|TLM.*|WX.*|ZIP.*)");
+        QRegularExpression re_mice("^[A-LP-Z0-9]{3}[L-Z0-9]{3}.?$"); // Mic-E Encoded Destination, 6-7 bytes
+        if (re.match(packet.m_to).hasMatch() || re_mice.match(packet.m_to).hasMatch())
```

Comment:
The `m_to` field is used to encode the latitude, message type, cardinal direction of lat and lon, as well as the longitude offset bit. 

I considered making it separate, which I'm happy to, as this might match things it should not.